.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/elixir:1.3.4
+      - image: redis:2.8.6
+    working_directory: /home/circleci/redbird
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - asdf-node-cache-{{ arch }}-{{ checksum ".tool-versions" }}
+      - restore_cache:
+          keys:
+          - dependency-cache-{{ arch }}-{{ .Branch }}-{{ checksum "mix.lock" }}
+          - dependency-cache-{{ arch }}-{{ .Branch }}
+          - dependency-cache-{{ arch }}
+      - run: mix local.hex --force && mix local.rebar --force && mix deps.get
+      - save_cache:
+          key: asdf-node-cache-{{ arch }}-{{ checksum ".tool-versions" }}
+          paths: "~/.asdf"
```

Comment:
I don't think this project uses `node` for anything, so you might be able to skip this.